src/lib/ride.ts

```diff
@@ -120,9 +120,10 @@ export async function onRideEnd(ride: RideModel) {
     throw new StravaNotConnected(
       'Not uploading - not authenticated with strava.',
     );
+  } else {
```

Comment:
I don't understand why the else is necessary, if there is no token the error above will throw and the function will return early.

Reply:
thats true, i read that as an alert rather than a throw. I'll revert this.